src/epp_topics/python_basics/dicts/objectives_materials.ipynb

```diff
@@ -79,6 +79,73 @@
     "    },\n",
     "    {\n",
     "        \"question\": (\n",
+    "            \"Which of the following types of objects can you use as dictionary keys?\"\n",
+    "        ),\n",
+    "        \"type\": \"many_choice\",\n",
+    "        \"answers\": [\n",
+    "            {\n",
+    "                \"answer\": \"\"\"A list.\"\"\",\n",
+    "                \"correct\": False,\n",
+    "                \"feedback\": \"Correct!\",\n",
+    "            },\n",
+    "            {\n",
+    "                \"answer\": \"\"\"A tuple.\"\"\",\n",
+    "                \"correct\": True,\n",
+    "                \"feedback\": \"Incorrect.\",\n",
+    "            },\n",
+    "            {\n",
+    "                \"answer\": \"\"\"An integer\"\"\",\n",
+    "                \"correct\": True,\n",
+    "                \"feedback\": \"\"\"Correct!\"\"\",\n",
+    "            },\n",
+    "            {\n",
+    "                \"answer\": \"\"\"A set.\"\"\",\n",
+    "                \"correct\": False,\n",
+    "                \"feedback\": \"\"\"Incorrect.\"\"\",\n",
+    "            },\n",
+    "            {\n",
+    "                \"answer\": \"\"\"A floating point number.\"\"\",\n",
+    "                \"correct\": True,\n",
+    "                \"feedback\": \"\"\"Correct, though not recommended due to floating point\n",
+    "                    precision issues.\"\"\",\n",
+    "            },\n",
+    "            {\n",
+    "                \"answer\": \"\"\"An Boolean.\"\"\",\n",
```

Comment:
A boolean, not An boolean

Reply:
https://en.wikipedia.org/wiki/George_Boole